src/Classes/Controllers/DisplayApplicantsController.php

```diff
@@ -36,7 +36,30 @@ public function __construct(PhpRenderer $renderer, ApplicantModel $applicantMode
      */
     public function __invoke(Request $request, Response $response, array $args)
     {
-        $args['data'] = $this->applicantModel->getAllApplicants();
+
+        $sortValue = $request->getQueryParam('sort');
+        $args['sort'] = $sortValue;
```

Comment:
Rather than adding your sort to the existing args array, you should create a new array that gets sent to your view (that will include your 'sort' and 'data') instead. Slims including examples take the $args array from the invoke method and pass it straight to the view, but this is bad practise in reality as it's very insecure.

Reply:
Created new $params array, used to store the sorting data. (Passed in the response)